about/ols-openshift-lightspeed-overview.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="openshift-lightspeed-overview"]
+include::_attributes/common-attributes.adoc[]
+= OpenShift Lightspeed overview
+:context: openshift-lightspeed-overview
+
+toc::[]
+
+{ols-official} provides intelligent, natural language processing capabilities designed to make Red Hat cloud-native application platforms easier to use for novice users and more efficient for experienced professionals. 
```

Comment:
done

Reply:
The team wanted to use the phrase natural language because that's the term often used with AI chat. We'll go with this for the early access program and see what additional feedback we receive